internal/bgp/frr/testdata/TestSingleAdvertisement.golden

```diff
@@ -45,6 +47,14 @@ router bgp 100
   address-family ipv4 unicast
     neighbor 10.2.2.254 activate
     neighbor 10.2.2.254 route-map 10.2.2.254-in in
-    network 172.16.1.10/24
     neighbor 10.2.2.254 route-map 10.2.2.254-out out
   exit-address-family
+  address-family ipv6 unicast
+    neighbor 10.2.2.254 activate
+    neighbor 10.2.2.254 route-map 10.2.2.254-in in
+    neighbor 10.2.2.254 route-map 10.2.2.254-out out
+  exit-address-family
```

Comment:
From looking at the `TestSingleAdvertisement` test, I think we shouldn't have this here.

Reply:
**Call the family:= assignment**
https://github.com/fedepaol/metallb/blob/b7ed3c93898d17165af9a3439ece45eb0c103673/internal/bgp/frr/frr.go#L272

**switch family:**
https://github.com/fedepaol/metallb/blob/b7ed3c93898d17165af9a3439ece45eb0c103673/internal/bgp/frr/frr.go#L292

Reply:
same